scripts/predicting_the_past_import.py

```diff
@@ -43,10 +43,16 @@ def import_pris(pris_link):
     """
     pris = pd.read_csv(pris_link,
                        delimiter=',',
-                       encoding='iso-8859-1'
+                       encoding='iso-8859-1',
+                       skiprows=20,
+                       usecols=[2, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]
```

Comment:
Pros of using numbers here: 
- It's compact
- Can reduce the total lines of code

Cons of using numbers here: 
- Users don't know what those columns mean without looking at the csv file.
- Inflexible (what happens if future versions of this dataset add a new column between two others? Reorder the columns?)

I would recommend doing the following: 
```python
usecols=['data1',
               'data2', 
               'data3',
               '...']
```
It uses more lines of code, but the meaning is very clear. It will also fix some of my other magic number complaints later on (I started making comments in a weird order, sorry).

Reply:
The spacing in my example did not work, obviously use pep8 spacing